packages/user/test/user.spec.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import { JupyterServer } from '@jupyterlab/testutils';
```

Comment:
Could you please drop the test part if none are implemented?

Reply:
What do you mean? removing the test completely (user.spec.ts file)?

Reply:
I suppose you should remove this file yes. We can add it back when there are actually tests.

Reply:
I can not remove the tests. The ensure repo script complains.